research-hub-web/src/app/app.component.html

```diff
@@ -1,301 +1,266 @@
+                    <span fxHide.gt-sm>{{ userInfo?.firstName?.charAt(0) + userInfo?.lastName?.charAt(0) }}</span>
                   </button>
-                </mat-menu>
-
-                <button 
-                  mat-button 
-                  fxHide.lt-md 
-                  routerLink="/userStudy">
-                    User Study
-                </button>
-
-                <button 
-                  mat-button 
-                  fxHide.lt-md 
-                  routerLink="/feedback">
-                    Feedback
-                </button>
-
-                <button 
-                  *ngIf="!authenticated"
-                  fxHide.lt-md 
-                  mat-button 
-                  (click)="loginService.doLogin()">
-                    Sign In
-                </button>
-
-                <button 
-                  class="button-menu" 
-                  *ngIf="authenticated" 
-                  mat-button 
-                  [matMenuTriggerFor]="userMenu">
-                  <mat-icon 
-                    matPrefix 
-                    style="margin-right: 1vh;" 
-                    fxHide.xs>
-                      person
-                  </mat-icon>
-                  <span 
-                    fxHide 
-                    fxShow.gt-md>
-                      {{ userInfo?.firstName + ' ' + userInfo?.lastName }}
-                  </span>
-                  <span 
-                    fxHide 
-                    fxShow.md>
-                      {{ userInfo?.firstName }}
-                  </span>
-                  <span fxHide.gt-sm>
-                      {{ userInfo?.firstName?.charAt(0) + userInfo?.lastName?.charAt(0) }}
-                  </span>
-                </button>
-
-                <mat-menu 
-                  #userMenu="matMenu" 
-                  [overlapTrigger]="false">
-                  <div class="user-detail-dropdown">
-                    <img src="assets/images/uoa-logo-blue.svg">
-
-                    <mat-list class="user-menu-list">
-                      <mat-list-item>
-                        <mat-icon mat-list-icon>person</mat-icon>
-                        <p mat-line>{{ userInfo?.firstName + ' ' + userInfo?.lastName }}</p>
-                      </mat-list-item>
-                      <mat-list-item>
-                        <mat-icon mat-list-icon>note</mat-icon>
-                        <p mat-line>{{ userInfo?.email }}</p>
-                      </mat-list-item>
-                    </mat-list>
-
-                    <div 
-                      fxLayout="row" 
-                      fxLayoutAlign="end">
-                      <button 
-                        mat-raised-button 
-                        class="sign-out-button"
-                        href="https://www.auckland.ac.nz/en/about-us/about-the-university/identity-and-access-management/single-sign-on/details.html">
+
+                  <!-- User Details -->
```

Comment:
I've put a note in the backlog to redesign the user details. It needs a refresh, but don't worry about it for now unless you have a free moment haha ;)